kamon-annotation/src/test/java/kamon/annotation/AnnotatedJavaClass.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * =========================================================================================
+ * Copyright © 2013-2015 the kamon project <http://kamon.io/>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ * =========================================================================================
+ */
+
+package kamon.annotation;
+
+@Metrics
+public class AnnotatedJavaClass {
+
+    public static String ID = "10";
+
+    @Trace("trace")
+    public static void trace() {}
+
+    @Trace("trace-with-segment")
+    @Segment(name = "segment", category = "segments", library = "segment")
+    public static void segment() {
```

Comment:
I'm not sure of what is the case of having both `@Trace` and `@Segment` in the same method, can you tell me a bit more about it?